unearthed-arcana/2015/20151207.xml

```diff
@@ -23,6 +23,84 @@
+			<p class="caption">Abyssal Arcana Spells</p>
+			<table>
+				<thead>
+					<tr><td>d6</td><td>1st Level</td><td>3rd Level</td><td>5th Level</td></tr>
+				</thead>
+				<tr><td>1</td><td><em>Dancing lights</em></td><td><em>Burning hands</em></td><td><em>Alter self</em></td></tr>
+				<tr><td>2</td><td><em>True strike</em></td><td><em>Charm person</em></td><td><em>Darkness</em></td></tr>
+				<tr><td>3</td><td><em>Light</em></td><td><em>Magic missle</em></td><td><em>Invisibility</em></td></tr>
+				<tr><td>4</td><td><em>Message</em></td><td><em>Cure wounds</em></td><td><em>Levitate</em></td></tr>
+				<tr><td>5</td><td><em>Spare the dying</em></td><td><em>Tasha's hideous laughter</em></td><td><em>Mirror image</em></td></tr>
+				<tr><td>6</td><td><em>Prestidigitation</em></td><td><em>Thunderwave</em></td><td><em>Spider climb</em></td></tr>
+			</table>
+		</description>
+		<sheet>
+			<description>Whenever you finish a long rest, roll on the Abyssal Arcana Spells table to determine your random cantrip, this cantrip can be cast at will. You lose cantrip previously granted by this feature when you finish a long rest. If you roll the same cantrip you gained at the end of your previous long rest, roll again until you get a different result.</description>
+			<description level="3">Whenever you finish a long rest, roll on the Abyssal Arcana Spells table to determine your random cantrip and 1st-level spell, rolling separately for each. You can cast cantrip at will. You can also cast a 1st-level spell once (if the rolled spell can be used with a higher level slots, you cast it at the 2nd-level). You lose cantrip and spell previously granted by this feature when you finish a long rest. If you roll the same cantrip or spell you gained at the end of your previous long rest, roll again until you get a different result.</description>
+			<description level="5">Whenever you finish a long rest, roll on the Abyssal Arcana Spells table to determine your random cantrip, and spells of 1st and 2nd level, rolling separately for each. You can cast cantrip at will. You can also cast each spell once (if the rolled spell of 1st-level can be used with a higher level slots, you cast it at the 2nd-level). You lose cantrip and spells previously granted by this feature when you finish a long rest. If you roll the same cantrip or spell you gained at the end of your previous long rest, roll again until you get a different result.</description>
+		</sheet>
+	</element>
+	<element name="Abyssal Fortitude" type="Racial Trait" source="Unearthed Arcana: That Old Black Magic" id="ID_WOTC_UA20151207_RACIAL_TRAIT_ABYSSAL_ABYSSAL_FORTITUDE">
+		<description>
+			<p>Your hit point maximum increases by half your level (minimum 1).</p>
+		</description>
+		<sheet>
+			<description>Your hit point maximum increases by half your level (minimum 1).</description>
+		</sheet>
```

Comment:
so there's two options we have when it comes to this, we can either keep it but use stat to make clear how much HP character gets or hide sheet description, i'm inclined to do the latter, just because i think that might confuse people in wanting to add these HP over already calculated HP.

Reply:
Hid the sheet description